blueocean-dashboard/src/main/js/components/testing/TestResults.jsx

```diff
@@ -141,7 +65,10 @@ export default class TestResult extends Component {
         // always show fixed, whether showing totals or the encouraging message
         if (fixed.length > 0) {
             fixedBlock = (<div className="test-result-block fixed-block">
-                <h4>{translation('rundetail.tests.results.fixed', { defaultValue: 'Fixed' })}</h4>
+                <h4>{translation('rundetail.testResults.results.fixed', {
```

Comment:
it's called "tests" elsewhere, but this was named to "testResults" - seems a bit odd? (and creates more translation rework)